src/volur/api/v1alpha1/client.py

```diff
@@ -111,3 +112,93 @@ async def generate_requests() -> (
                 code=code,
                 message=message,
             )
+
+    async def upload_products_information(
+        self: "VolurApiAsyncClient",
+        products: AsyncIterator[product_pb2.Product],
+    ) -> Status:
+        """Uploads Products Information to the Völur platform using the Völur
+        API.
+
+        This method is using a source to get the products data and then send
+        it to the Völur API. This method is asynchronous and will return a
+        status of the operation.
+
+        Args:
+            products: a source of materials data to be uploaded to the Völur
```

Comment:
Feel free to resolve

I would be picky and be careful with usage of a word `source`, would rather rephrase it this way:

```suggestion
            products: an iterable of Product protos to be uploaded via API
```

Reply:
Updated